atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -207,7 +207,7 @@ def run(self):
             self.log.debug("annotations: %r", annotations)
             raise
 
-        labels = self.make_labels()
+        labels = self.make_labels(metadata)
```

Comment:
That's possible, although it would complicate `sendmail` plugin a bit more - it would duplicate actions `store_metadata` does to fetch koji task id.